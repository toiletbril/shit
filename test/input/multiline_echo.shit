echo \
привет мир! \
hello sailor!
