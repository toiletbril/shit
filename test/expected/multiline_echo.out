[CompoundList]
  [CompoundListCondition, None]
    [SimpleCommand "echo привет мир! hello sailor!"]
[Escape Map
  00000100 00000000 00000000 00001000 
]
привет мир! hello sailor!
[Code 0]
